src/java/io/bazel/rulesscala/scalac/ScalacWorker.java

```diff
@@ -21,17 +20,6 @@
 class ScalacWorker implements Worker.Interface {
   private static boolean isWindows = System.getProperty("os.name").toLowerCase().contains("windows");
 
-  /** This is the reporter field for scalac, which we want to access */
-  private static Field reporterField;
-
-  static {
-    try {
-      reporterField = Driver.class.getDeclaredField("reporter"); // NoSuchFieldException
```

Comment:
This field could be maintained to get access to the `ReportableMainClass` implementation on our side, but I do think this would be unnecessary added complexity.